pysat/instruments/jro_isr.py

```diff
@@ -257,4 +260,31 @@ def calc_measurement_loc(self):
         self.data = self.data.assign(lat_key=gdlat, lon_key=gdlon)
         self.data.rename({"lat_key":lat_key, "lon_key":lon_key}, inplace=True)
 
+        # Add metadata for the new data values
+        bm_label = "Beam {:d} ".format(dd)
+        self.meta.__setitem__(lat_key, {self.meta.units_label: 'degrees',
```

Comment:
self.meta[lat_key] = {stuff} should work here, if it doesn't let me know.

Reply:
May or may not work, I couldn't find an example so I opened up the class to figure it out.  I'll try your suggestion and let you know.